internal/idp_test/azuread_util_test.go

```diff
@@ -0,0 +1,243 @@
+package idp_test
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/elimity-com/scim"
+	"github.com/elimity-com/scim/errors"
+	"github.com/elimity-com/scim/optional"
+	"github.com/elimity-com/scim/schema"
+	"github.com/scim2/filter-parser/v2"
+)
+
+func newAzureADTestServer() scim.Server {
+	return scim.Server{
+		Config: scim.ServiceProviderConfig{
+			MaxResults: 20,
+		},
+		ResourceTypes: []scim.ResourceType{
+			{
+				ID:          optional.NewString("User"),
+				Name:        "User",
+				Endpoint:    "/Users",
+				Description: optional.NewString("User Account"),
+				Schema:      schema.CoreUserSchema(),
+				SchemaExtensions: []scim.SchemaExtension{
+					{Schema: schema.ExtensionEnterpriseUser()},
+				},
+				Handler: azureADUserResourceHandler{},
+			},
+
+			{
+				ID:          optional.NewString("Group"),
+				Name:        "Group",
+				Endpoint:    "/Groups",
+				Description: optional.NewString("Group"),
+				Schema:      schema.CoreGroupSchema(),
+				Handler:     azureADGroupResourceHandler{},
+			},
+		},
+	}
+}
+
+type azureADGroupResourceHandler struct{}
+
+func (a azureADGroupResourceHandler) Create(r *http.Request, attributes scim.ResourceAttributes) (scim.Resource, error) {
+	created, _ := time.Parse(time.RFC3339, "2018-03-27T19:59:26.000Z")
+	return scim.Resource{
+		ID:         "927fa2c08dcb4a7fae9e",
+		ExternalID: optional.NewString(attributes["externalId"].(string)),
+		Attributes: attributes,
+		Meta: scim.Meta{
+			Created:      &created,
+			LastModified: &created,
+		},
+	}, nil
+}
+
+func (a azureADGroupResourceHandler) Delete(r *http.Request, id string) error {
+	return errors.ScimError{
+		Status: http.StatusNotImplemented,
+	}
+}
+
+func (a azureADGroupResourceHandler) Get(r *http.Request, id string) (scim.Resource, error) {
+	created, _ := time.Parse(time.RFC3339, "2018-03-27T19:59:26.000Z")
```

Comment:
Wouldn't it be simpler to just use `time.Date` here? Same comment for the other occurrences in this file.